robosuite/scripts/demo_device_control.py

```diff
@@ -151,6 +153,8 @@
         ignore_done=True,
         use_camera_obs=False,
         gripper_visualizations=True,
+        #render_collision_mesh=True,
```

Comment:
are these necessary?

Reply:
I actually did not touch this file - I don't know why there is a difference.